### ISSUE TITLE
White pages and high cpu

### ISSUE THREAD

**Issue by Leen15** [gioxx/MarvellousSuspender#171]
I cannot say if it's a chrome update or the extension, but since about 3-6 days I have very weird behaviour.
Most of the suspended pages has only a white page when I can back to that tab, and I can see the suspended page only after 30-60 seconds.
In the same way, I have a 100% cpu on chrome main process (not the extension one).
If I restart chrome I can come back to normal cpu load, but after a couple of suspend / not suspend actions, I've the same 100% cpu.

Anybody else notice this?

Thanks

 * Extension version: 7.1.6.2
 * Browser name & version: Chrome 99.0.4844.84
 * Operating system & version: Mac OS 12.2

**Comment by pressRtowin** [comment]
Having the same issue now on Windows 10 ever since updating chrome 5 days or so ago.

**Comment by Technetium1** [comment]
Can you confirm this is an issue after updating Chrome? 
Probably a duplicate of https://github.com/gioxx/MarvellousSuspender/issues/166

https://chromereleases.googleblog.com/2022/04/stable-channel-update-for-chrome-os.html

**Comment by MonsterNya** [comment]
The problem is exactly the same for me. not the first day.

p.s. I don't follow chrome updates, upd automatically.

Extension version: 7.1.6.2
Browser name & version: Chrome 100.0.4896.60
Operating system & version: Windows 10 1703 (15063.332)

**Comment by rqen** [comment]
I got the same issue with the page being white, tho it doesn't seem to make my CPU go bananas.
But there doesn't seem to be consistency in the page going white, as (more or less) every second click has the expected behaviour.

Trying to share a screen recording, of me clicking between tabs(1 active and 2 suspended):
https://user-images.githubusercontent.com/1307521/161561713-b1d23bea-0244-4076-a40b-81359e025e5e.mov


Extension version: 7.1.6.2
Operating system & version: MacOS Monterey 12.3.1
Browser name & version: Chrome 100.0.4896.60

**Comment by ndom91** [comment]
Also having the white page issue. Refreshing the page sometimes brings it back to the suspended marvelous suspender page, sometimes not. CPU doesn't go wild for me though.

Extension: `7.1.6.2`
OS: `Arch Linux 5.16-5`
Browser: `Brave 1.36.119 Chromium: 99.0.4844.83 (Official Build) (64-bit)`

**Comment by mboveiri** [comment]
same with latest chrome stable version

**Comment by rqen** [comment]
Seems to be (somewhat) fixed with the latest Chrome version 100.0.4896.75.
It still flickers a bit, and feels slow, when you go to a suspended tab, but the behaviour is more as expected :)

**Comment by Leen15** [comment]
I found a workaround about the white pages on suspended tabs.
I disabled this option:
"Apply Chrome's built-in memory-saving when suspending"
in the extension options page and it seems to work much better (you have to suspend again the tabs after changed the flag).

About the CPU issue, a colleague has the same issue and he's not using this extension so I guess it's not related to it.

**Comment by mikefirefly** [comment]
> I found a workaround about the white pages on suspended tabs. I disabled this option: "Apply Chrome's built-in memory-saving when suspending" in the extension options page and it seems to work much better (you have to suspend again the tabs after changed the flag).

Doesn't work for me - Brave Version 1.37.111 Chromium: 100.0.4896.79 (Official Build) (64-bit)
After disabling this option I **always** end up with the white page.

**Comment by kraivux** [comment]
Using Edge 100.0.1185.29 (Official build) (64-bit), white screen but none of the other noted issues. Switching to another suspended tab and back would cause the familiar suspended page to load.

I thought maybe the problem might be with loading 'suspended.html' so I enabled "Extensions on edge:// URLs" in edge://flags and it seems to have solved the issue for me. It hasn't been more than ten minutes so it's quite possible restarting to enable the setting properly did it (though previous restarts did nothing) but if anyone using Edge (or if any other Chromium browser having the same issue has a similar setting) can verify it would be much appreciated.

**Comment by sivan** [comment]
Looks fixed after upgrade Chrome to v100.0.4896.88

**Comment by pressRtowin** [comment]
> Looks fixed after upgrade Chrome to v100.0.4896.88

I thought so too, but then it started doing it again just within the last 24 hours (and I intentionally haven't installed the latest update that Chrome has been prompting me about).

**Comment by pbowyer** [comment]
Still doing the white tab thing for me too. I have to click on the suspended tab I want (white page), change to another suspended tab (another white page), and then go back to the one I want (now after a second it changes from a white page to the "Marvellous Suspender" suspended tab screen).

The Marvellous Suspender v7.1.6.2 with "Apply Chrome's built-in memory-saving when suspending" turned on (off didn't fix it)
Chrome 100.0.4896.88 (Official Build) (64-bit)

**Comment by pbowyer** [comment]
This issue is a duplicate of https://github.com/gioxx/MarvellousSuspender/issues/161

There's a fix in GitHub but it's not deployed to Chrome extension directory yet.
